test/scanner_test.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+INPUT_FILES="scanner/*_input.txt"
```

Comment:
yep, that's just a placeholder since the tests error if we don't have any. we should delete these when we write better tests.